### ISSUE TITLE
Jack-in uses cmd.exe on Windows 10 with Powershell-version of leiningen

### ISSUE THREAD

**Issue by rogererens** [BetterThanTomorrow/calva#428]
I'm having this issue that the Jack-in task seems to insist on using C:\WINDOWS\system32\cmd.exe to run lein, whereas I have the Powershell version of leiningen, lein.ps1, in my PATH.

I set powershell to be my Default Shell.

Still, this is the task that ctrl-alt-c ctrl-alt-j executes:
`Executing task: C:\WINDOWS\system32\cmd.exe /d /c lein update-in :dependencies conj [nrepl"0.6.0"] -- update-in :plugins conj [cider/cider-nrepl"0.22.4"] -- update-in [:repl-options^ :nrepl-middleware] conj '["cider.nrepl/cider-middleware"]' -- repl :headless <`

Resulting in this output:
`'lein' is not recognized as an internal or external command,
operable program or batch file.
The terminal process terminated with exit code: 1

Terminal will be reused by tasks, press any key to close it.`

I can apparently work-around it by downloading lein.bat into my PATH.

**Comment by PEZ** [comment]
Yes, the Leiningen jack-in only supports the cmd.exe way. I'm not sure what it would entail to also support the Powershell version. (Which I didn't know existed, even.  😄 ) Judging from the struggle it was to get it working for tools.deps, I sort of suspect that this would be quite some work as well.

**Comment by cfehse** [comment]
@rogererens 

Did you test putting `lein.bat` in our PATH as well, and if so, did it work?

**Comment by rogererens** [comment]
@cfehse Yes, and yes (AFAICT).

The powershell version of Leiningen is a work in progress. Personally, I'm trying to migrate away from cmd to powershell, as it has a slightly more *nix-y command line interface (ls, rm), which is why I was trying to avoid lein.bat.

**Comment by cfehse** [comment]
@rogererens 

I understand your intention to use the powershell version. So we marked this issue as enhancement and leave it open but you could work with calva if you would use the bat file. That's better than nothing for now. *gg*
